crates/typst-library/src/model/par.rs

```diff
@@ -431,12 +431,12 @@ pub struct ParLine {
     #[default]
     pub number_clearance: Smart<Length>,
 
-    /// Controls when to reset line numbering.
+    /// 行番号をリセットするタイミングを制御する。
     ///
-    /// _Note:_ The line numbering scope must be uniform across each page run (a
-    /// page run is a sequence of pages without an explicit pagebreak in
-    /// between). For this reason, set rules for it should be defined before any
-    /// page content, typically at the very start of the document.
+    /// _注意:_ 行番号のスコープは、
+    /// ページラン（改ページが明示的に挿入されていない連続したページ）内で統一されている必要があります。
+    /// そのため、setルールによる設定は、
+    /// ページコンテンツの前、通常は文書の最初などで定義する必要があります。
```

Comment:
「必要があります。」で終わる文が続いているため、2文目を少し変えてみるとよいかもしれません。
この部分の原文ではmust beとshould beになっています。

```suggestion
    /// _注意:_ 行番号のスコープは、
    /// ページラン（改ページが明示的に挿入されていない連続したページ）内で統一されている必要があります。
    /// そのため、setルールによる設定は、
    /// ページコンテンツの前、通常は文書の最初などで定義することが望ましいです。